api/app/Entity/SocialAccount.php

```diff
@@ -7,6 +7,10 @@
 class SocialAccount extends Model
 {
     public const GOOGLE_SERVICE_ID = 1001;
+    protected $primaryKey = null;
+    public $incrementing = false;
```

Comment:
Is it really necessary?

Reply:
We don't have the PK on that table and when we use firstOrNew and save it tries to insert field with the id